src/vs/platform/extensionManagement/node/extensionManagementService.ts

```diff
@@ -123,6 +123,13 @@ export class ExtensionManagementService implements IExtensionManagementService {
 
 				this._onInstallExtension.fire({ identifier, zipPath });
 
+				// Until there's a gallery for SQL Ops Studio, skip retrieving the metadata from the gallery
```

Comment:
Could you please add a line `// {{SQL CARBON EDIT}}` here on line 126?  We use this comment to identify any changes we make to the VS Code source code under the `src/vs` directory.